table/column.go

```diff
@@ -70,6 +70,16 @@ func FindCol(cols []*Column, name string) *Column {
 	return nil
 }
 
+// FindColInfo finds column in cols by name.
+func FindColInfo(cols []*model.ColumnInfo, name string) bool {
+	for _, col := range cols {
+		if strings.EqualFold(col.Name.O, name) {
```

Comment:
You pass the `name` by `key.Column.Name.L`, but here check with `col.Name.O`.

Reply:
done